### ISSUE TITLE
about batch-size and replay-batch-size

### ISSUE THREAD

**Issue by massizhi** [hongjoon0805/SS-IL-Official#11]
Hello!
When I saw your loss calculation formula(L CE-SS), I thought you would calculate the sum of the average losses of the two loss function. However, in the code, you use batch-size and replay-batch-size to adjust the weights of the old and new class losses ( loss_CE = (loss_CE_curr + loss_CE_prev) / (batch_size + replay_size) ). Did you view the ratio between batch-size and replay-batch-size as a hyperparameter to adjust in your experiments?
Thank you!
